about.html

```diff
@@ -12,53 +12,64 @@
 </head>
 
 <body>
+  <div class="container">
+  <div class="navigation-bar">
+    <div class="lambda-logo">
+      <h1>ambda</h1>
+    </div>
```

Comment:
This navigation/header area is a perfect place to use semantic HTML to do the layout of the top part of the webpage.  Instead of using divs, you can use header and nav elements.